tests/TestCase.php

```diff
@@ -11,6 +11,30 @@
 
 abstract class TestCase extends BaseTestCase
 {
+    public function expectException($exception)
+    {
+        if (method_exists('PHPUnit\Framework\TestCase', 'expectException')) {
```

Comment:
This can be `BaseTestCase::class` in all three cases. Alternatively, `get_parent_class()` could also be used (https://3v4l.org/Wh5YO) to keep things flexible, but I can't imagine we're going to change the parent class.